test/plugins/builds.test.js

```diff
@@ -107,10 +109,12 @@ describe('builds plugin test', () => {
                     headers: {
                         'x-foo': 'bar'
                     },
-                    credentials: {
-                        username: mockBuildID,
-                        scope: ['user']
-                    },
+                    auth: {
+                        strategy: 'token',
+                        credentials: {
```

Comment:
Is `credentials` is part of `auth` now, and we need to explicitly state which `strategy`? 🤔

Reply:
Yes, thats, correct